rust-module/mfrc522.rs

```diff
@@ -80,6 +93,11 @@ impl FileOperations for Mfrc522FileOps {
             Ok(cmd) => cmd,
         };
 
+        if unsafe { SPI_DEVICE.is_none() } {
```

Comment:
Is it unsafe to just access it non-mutably? Can't we just do
```suggestion
        if SPI_DEVICE.is_none() {
```
?